app/src/Controller/Robots.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Dappur\Controller;
+
+use Dappur\Controller\Controller as Controller;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+use Steein\Robots\Robots as R;
+use Steein\Robots\RobotsInterface;
+
+class Robots extends Controller
+{
+    public function view(Request $request, Response $response)
```

Comment:
__[PHPMD]__ <sub>posted by Sider</sub>

The method view() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.